### ISSUE TITLE
Overloads for saga correlation methods

### ISSUE THREAD

**Issue by seankearon** [rebus-org/Rebus#883]
When we correlate messages that participate in sagas against their data, we use an Expression as the second parameter.  Like this example from the [wiki](https://github.com/rebus-org/Rebus/wiki/Coordinating-stuff-that-happens-over-time):

``` c#
    protected override void CorrelateMessages(ICorrelationConfig<CollectLegalInfoSagaData> config)
    {
        config.Correlate<CustomerCreated>(m => m.CustomerId, d => d.CrmCustomerId);
        config.Correlate<LegalInfoAcquiredInFirstSystem>(m => m.CorrId, d => d.CrmCustomerId);
        config.Correlate<LegalInfoAcquiredInSecondSystem>(m => m.CorrId, d => d.CrmCustomerId);
    }
```

I think this parameter is only used to identify the name of the property in the SagaData. 

Would it be sensible to have an overload for the second parameter that allowed us to pass a string?  For example:

``` c#
    protected override void CorrelateMessages(ICorrelationConfig<CollectLegalInfoSagaData> config)
    {
        config.Correlate<CustomerCreated>(m => m.CustomerId, nameof(CollectLegalInfoSagaData.CrmCustomerId));
    }
```
My motivation is to make it easier to use a saga from another language, F# in this case.

I think this would also apply to the other methods in [ICorrelationConfig.cs](https://github.com/rebus-org/Rebus/blob/master/Rebus/Sagas/ICorrelationConfig.cs) also.

**Comment by mookid8000** [comment]
Yes, that would actually be great 🙂

**Comment by seankearon** [comment]
Great!   How does [this](https://github.com/rebus-org/Rebus/pull/884) look?

**Comment by mookid8000** [comment]
Fixed in Rebus 6.3.0 🤘
